lib/gui/app/pages/main/templates/main.tpl.html

```diff
@@ -64,13 +64,16 @@
           <div class="step-selection-text"
             ng-class="{
               'text-disabled': main.shouldDriveStepBeDisabled()
-            }"
-            uib-tooltip="{{ main.selection.getDrive().description }} ({{ main.selection.getDrive().displayName }})">
-            <span class="step-drive step-name">
+            }">
+            <span class="drive-step step-name"
+              uib-tooltip="{{ main.selection.getDrive().description }} ({{ main.selection.getDrive().displayName }})">
               <!-- middleEllipses errors on undefined, therefore fallback to empty string -->
               {{ (main.selection.getDrive().description || "") | middleEllipses:11 }}
             </span>
             <span class="step-drive step-size">{{ main.selection.getDrive().size | closestUnit }}</span>
+            <span class="step-drive step-warning glyphicon glyphicon-exclamation-sign"
+              uib-tooltip="{{ main.constraints.getDriveImageCompatibilityStatuses(main.selection.getDrive(), main.selection.getImage())[0].message }}"
```

Comment:
Will this code still be run if `ng-show` is `false`?

Reply:
I checked this, and it does – luckily the fix is simple and I'll open a new PR for that.

Reply:
https://github.com/resin-io/etcher/pull/2021